internal/model/plugin.go

```diff
@@ -7,13 +7,19 @@ import (
 	mattermostModel "github.com/mattermost/mattermost-server/model"
 )
 
+// PluginSignatures represents Mattermost plugin signatures
```

Comment:
nit: this comment, while technically accurate, seems unhelpful. Perhaps:
```suggestion
// PluginSignatures is a list of public key signatures and the corresponding public key hash for use in verifying a plugin downloaded from the marketplace.
//
// A plugin may have multiple signatures if it is signed by multiple public keys known to be trusted by different versions of the Mattermost server.
```